L1Trigger/TrackFindingTMTT/src/Histos.cc

```diff
@@ -0,0 +1,1776 @@
+#include "L1Trigger/TrackFindingTMTT/interface/Histos.h"
+#include "L1Trigger/TrackFindingTMTT/interface/InputData.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Sector.h"
+#include "L1Trigger/TrackFindingTMTT/interface/HTrphi.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Make3Dtracks.h"
+#include "L1Trigger/TrackFindingTMTT/interface/TrkRZfilter.h"
+#include "L1Trigger/TrackFindingTMTT/interface/L1fittedTrack.h"
+#include "L1Trigger/TrackFindingTMTT/interface/Utility.h"
+#include "L1Trigger/TrackFindingTMTT/interface/PrintL1trk.h"
+
+#include "DataFormats/Math/interface/deltaPhi.h"
+#include "DataFormats/Math/interface/deltaR.h"
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include "TH1F.h"
+#include "TH2F.h"
+#include "TH2Poly.h"
+#include "TF1.h"
+#include "TPad.h"
+#include "TProfile.h"
+#include "TGraphAsymmErrors.h"
+#include "TGraph.h"
+#include "TEfficiency.h"
+
+#include <algorithm>
+#include <array>
+#include <unordered_set>
+#include <list>
+#include <sstream>
+#include <memory>
+#include <mutex>
+
+using namespace std;
+
+namespace tmtt {
+
+  namespace {
+    std::once_flag printOnce;
```

Comment:
move this to a `static` variable inside `Histos::fillTrackCands` ?

Reply:
done